dependencies.gradle

```diff
@@ -0,0 +1,12 @@
+/*
+ * Build dependencies
+ */
+ext.androidCompileSdkVersion = 26
+ext.androidMinSdkVersion = 19
+
+ext.androidBuildToolsVersion = '26.0.2'
```

Comment:
I actually think you don't need to specify the build tools version anymore. But this is fine, too.